docs/2.getting-started.md

```diff
@@ -1,12 +1,153 @@
 # 2. Getting Started
 
-## Installation Options
+## Installation
```

Comment:
Personally I'd make this a separate `INSTALLATION.md` doc, but because the doc-files are `#` prefixed, I didnt want to blow up the diff here.

Reply:
We can update that part after the package is available on Packagist.